services/s3control/src/main/resources/software/amazon/awssdk/services/s3control/execution.interceptors

```diff
@@ -1,2 +1,3 @@
+software.amazon.awssdk.services.s3.internal.handlers.DisableDoubleUrlEncodingInterceptor
```

Comment:
This makes it a protected API, not an internal API. If it's simple, can we just copy it?

Reply:
I discussed with Zoe and this was the solution we agreed on yesterday. You have a valid argument, but also S3 doesn't necessarily distinguish between S3 data plane and control plane API calls in terms on how they receive and process requests. Also we already have a dependency on S3. 

However I think I can be persuaded to copy it since it IS very small and dependencies are potentially expensive in the future...

Reply:
I think if we're going to share it and make it protected we should move it to a place that we can get to it without needing to go through the S3 module. Maybe in the `auth` module, since that's where it's used? We'd want to pick a more specific name as well: `DisableDoubleUrlEncodingDuringSigningInterceptor`?

I'm still on "team just copy it", but you and Zoe outnumber me :).